### DISCUSSION TITLE
Is this project accepting pull requests

### DISCUSSION THREAD

**Discussion by trampster** [dotnet/maui#365] [Q&A]
Can you please clarify if the Maui project is currently accepting pull requests.

There seems to be mixed messages:

The readme says this under how to contribute:
> Some of the best ways to contribute are to try things out, file issues, join in design conversations, and make pull-requests.

And @davidortinau made this statement about linux community contributions
>The signal here is meant to be that Linux is a platform on our radar, it is not part of our current scope of work for .NET 6, and we are open to collaboration in the meantime.

However if you make a pull request you get the following populated into the description box:
> "We are currently not accepting Pull Requests for .NET MAUI. We will continue to update this repository over the next couple of months and should be ready for PRs in early 2021."

If the project is not accepting pull requests I suggest updating the readme to say so. Otherwise people will waste time creating pull requests only to find out when they submit them that the project isn't accepting them. (this is what happened to me)

If the project is accepting pull requests then I suggest changing the pull request template to remove the text which says it isn't.

**Comment by davidortinau** [comment]
Thanks for asking, we are now! https://github.com/dotnet/maui/blob/main/.github/PULL_REQUEST_TEMPLATE.md has been updated to reflect this.
